lib/dpd_api/requests/store_orders.rb

```diff
@@ -71,6 +71,8 @@ def xml_attributes
         attrs[:notification_channel] ||= 1
         attrs[:notification_value] ||= attrs[:shipper_address][:email]
         attrs[:notification_locale] ||= "EN"
+        attrs.fetch(:notification_rule, nil)
```

Comment:
did you want to remove this line after we chatted about it on Friday?

or if we continue with the odd "accessing fields in the code as an odd form of documentation" pattern I guess it would be this:

```suggestion
        attrs[:notification_rule]
```

IMO the best thing would be to strip these out and document the interface properly in the readme, as it's already caused both of us some confusion trying to work out whether these lines are relevant or not 😄 but that's not super urgent

Reply:
I'll drop the optional ones and add them in the readme. Potentially there's some usefulness to leaving the "mandatory" ones to be fetched and fail if missing. So I'll leave those in.